postgres-appliance/configure_spilo.py

```diff
@@ -377,6 +377,15 @@ def get_placeholders(provider):
     placeholders.setdefault('CLONE_TARGET_TIME', '')
     placeholders.setdefault('CLONE_TARGET_INCLUSIVE', True)
 
+    placeholders.setdefault('LOG_SHIP_SCHEDULE', '00 02 * * *')
```

Comment:
2 hours UTC? Why?

Reply:
selected simply as one hour later after the wal-e default time

Reply:
it should be `1 0 * * *`